all-requirements.txt

```diff
@@ -3,5 +3,9 @@
 # For simpl.rest
 bottle==0.12.8
 
+# For simpl.server
+eventlet==0.17.4
+tornado==4.2
```

Comment:
I know the pattern predates this PR, but would it make more sense for module-specific requirements to be listed in a module-specific requirements file?